sematic/db/migrations/20230609124213_pre_alembic_parity.sql

```diff
@@ -0,0 +1,47 @@
+-- migrate:up
+
+UPDATE runs SET function_path = '' WHERE function_path IS NULL;
+
+ALTER TABLE artifacts ALTER COLUMN type_serialization SET NOT NULL;
+ALTER TABLE edges ALTER COLUMN artifact_id SET NOT NULL;
+ALTER TABLE jobs ALTER COLUMN created_at SET NOT NULL;
+ALTER TABLE jobs ALTER COLUMN updated_at SET NOT NULL;
+ALTER TABLE notes ALTER COLUMN created_at SET NOT NULL;
+ALTER TABLE notes ALTER COLUMN updated_at SET NOT NULL;
+ALTER TABLE runs ALTER COLUMN function_path SET NOT NULL;
+ALTER TABLE runs ALTER COLUMN root_id SET NOT NULL;
+ALTER TABLE runs ALTER COLUMN tags SET NOT NULL;
+ALTER TABLE runs ALTER COLUMN source_code SET NOT NULL;
+
+ALTER TABLE edges ADD CONSTRAINT edges_destination_run_id_fkey FOREIGN KEY (destination_run_id) REFERENCES runs(id);
+ALTER TABLE edges ADD CONSTRAINT edges_source_run_id_fkey FOREIGN KEY (source_run_id) REFERENCES runs(id);
+
+ALTER TABLE metric_values ADD CONSTRAINT metric_values_metric_id_fkey FOREIGN KEY (metric_id) REFERENCES metric_labels(metric_id);
+ALTER TABLE runs ADD CONSTRAINT runs_root_id_fkey FOREIGN KEY (root_id) REFERENCES runs(id);
+
+ALTER INDEX jobs_run_id RENAME TO ix_jobs_run_id;
```

Comment:
alembic relies on a naming convention in order to associate specific indexes, foreign keys, unique constraints, etc. with their definitions in the sqlalchemy models.  i'm sure we have many such mismatches, but for whatever reason, these 3 indices were the only ones that alembic didn't like (it wanted to drop them and recreate them with the new name).  so i've just renamed them instead.